src/main/java/no/nav/tag/sykefravarsstatistikk/api/domene/statistikk/SykefraværsstatistikkLand.java

```diff
@@ -0,0 +1,17 @@
+package no.nav.tag.sykefravarsstatistikk.api.domene.statistikk;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import no.nav.tag.sykefravarsstatistikk.api.domene.virksomhetsklassifikasjoner.Virksomhetsklassifikasjon;
+
+import java.math.BigDecimal;
+
+@Data
+@AllArgsConstructor
+public class SykefraværsstatistikkLand implements Sykefraværsstatistikk {
```

Comment:
Er denne klassen egentlig en del av vårt domene? Jeg ser på denne klassen bare som et overføringsobjekt vi bruker når vi laster over fra DVH, og som aldri skal ha noe logikk. Sånn sett kan den bare legges i provisjonering-mappen? Da blir "Sammenligning" og "Sykefraværsprosent" de eneste domeneobjektene som har med statistikk å gjøre - det synes jeg er riktig, for det er det eneste vi tilbyr som tjeneste.

Det er veldig åpent for debatt det her, altså :P Jeg er ikke altfor selvsikker på DDD ennå.. Vi kan jo også diskutere å gå over til vanlig feature-basert mappestruktur, hvis det blir vanskelig å vedlikeholde den domene-mappen.

Reply:
Er ikke noe DDD entusiast heller, men det du skrev gir mening egentlig. Disse objektene er en del av datavarehus sin domene, ikke vår. Burde flyttes et annet sted.